training/heterogeneous-clusters/pt.grpc.sagemaker/README.md

```diff
@@ -0,0 +1,6 @@
+# SageMaker heterogeneous Training ("Hetero") - Pytorch example (MNIST Dataset)
```

Comment:
Copy content to an index.rst and link to your index.rst page in the upstream folder.